### PR TITLE
support micro-profile-sureness examples

### PR DESCRIPTION
**Pull request by CharlieXCL**
support example use sureness protect microprofile- https://start.microprofile.io/

### CODE REVIEW

File: samples/micro-profile-sureness/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# MicroProfile generated Application
+
+## Introduction
+
+MicroProfile Starter has generated this MicroProfile application for you.
+
+The generation of the executable jar file can be performed by issuing the following command
+
+    mvn clean package
```
Comment (tomsun28) [inline samples/micro-profile-sureness/README.md:9]:
```suggestion
    mvn clean package     
    mvn clean install -Pwildfly 
```

File: samples/micro-profile-sureness/pom.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    <id>jboss-public-repository</id>
+                    <name>JBoss Public Maven Repository Group</name>
+                    <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+                </repository>
+                <repository>
+                    <id>Red Hat GA</id>
+                    <name>Red Hat GA</name>
+                    <url>https://maven.repository.redhat.com/ga/</url>
+                </repository>
+            </repositories>
```
Comment (tomsun28) [inline samples/micro-profile-sureness/pom.xml:90]: rehat or jboos repository network is too slow

File: samples/micro-profile-sureness/src/main/webapp/WEB-INF/sureness.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+## -- sureness.yml document dataSource-- ##
```
Comment (tomsun28) [inline samples/micro-profile-sureness/src/main/webapp/WEB-INF/sureness.yml:1]: there are two same sureness.yml

File: samples/micro-profile-sureness/pom.xml (hunk 2 above)
Comment (CharlieXCL) [reply samples/micro-profile-sureness/pom.xml:90]: the repository is need

**Comment by tomsun28** [comment]
LGTM thanks.

**Pull request merged**